reference/api/documents.mdx

```diff
@@ -248,6 +248,45 @@ An array of documents. Each document is represented as a JSON object.
 ]
 ```
 
+#### `_vectors`
+
+`_vectors` is a special document attribute containing vector embeddings for AI-powered search. It accepts different types of values.
+
+`_vectors` may be an array of numbers:
+
+```json
+[
+  {
+    "id": 452,
+    "title": "Female Trouble",
+    "overview": "Delinquent high school student Dawn Davenport runs away from home and embarks upon a life of crime.",
+    "_vectors": [0.0, 0.1, 0.003, 0.75]
```

Comment:
Don't we want to promote this format instead?
`_vectors: { default: [...] } }

@dureuill

Reply:
Hello @curquiza 

this format is incorrect and will not work as soon as an embedder config has actually been declared.

See my comments below